tests/unit/mesh/data/test_gridfromoptions.cxx

```diff
@@ -11,240 +11,278 @@
 #include <string>
 #include <vector>
 
-/// Global mesh
-namespace bout {
-namespace globals {
-extern Mesh* mesh;
-} // namespace globals
-} // namespace bout
-
 // The unit tests use the global mesh
 using namespace bout::globals;
 
 class GridFromOptionsTest : public FakeMeshFixture {
 public:
-  GridFromOptionsTest() : FakeMeshFixture(), options(), griddata(&options) {
+  GridFromOptionsTest() : FakeMeshFixture(), options(), griddata(nullptr) {
```

Comment:
This now uses its own `mesh_from_options` -- could remove the inheritance from `FakeMeshFixture`? (would need to be replaced with inheritance from `::testing::Test`)